app/containers/VolunteerApprovalContainer/VolunteerApprovalContainer.js

```diff
@@ -0,0 +1,22 @@
+import React, { PropTypes, Component } from 'react'
+import VolunteerApproval from "../../components/VolunteerApproval/VolunteerApproval";
+import { connect } from 'react-redux'
+
+class VolunteerApprovalContainer extends Component {
+
+  // componentDidMount() {
+  //   this.props.dispatch(
+  //     fetchFeaturedEvents()
+  //   );
+  // }
```

Comment:
check diffs before committing for any unnecessary lines, so diffs are smaller and you don't commit things that are not needed